### ISSUE TITLE
Composer

### ISSUE THREAD

**Issue by jaroslawziolkowski** [garytong/quicbooks_sdk_psr0#1]
Have you this repository available by composer?

**Comment by garytong** [comment]
@jaroslawziolkowski No; this repo hasn't been used in a while and is not fully tested. I would not recommend using it in production. I may work on a PSR-4 version though.
